DESCRIPTION

```diff
@@ -43,15 +43,16 @@ Suggests:
     testthat (>= 3.0.4),
     withr (>= 2.0.0)
 VignetteBuilder:
-    knitr
+    knitr,
+    rmarkdown
 RdMacros:
     lifecycle
 Config/Needs/verdepcheck: mllg/checkmate, tidyverse/dplyr,
     r-lib/lifecycle, tidyverse/lubridate, tidyverse/magrittr, r-lib/rlang,
     tidyverse/stringr, tidyverse/tibble, tidyverse/tidyr,
     yaml=vubiostat/r-yaml, gowerc/diffdf, yihui/knitr, rstudio/rmarkdown,
     r-lib/testthat, r-lib/withr
-Config/Needs/website: insightsengineering/nesttemplate
+Config/Needs/website: rstudio/rmarkdown, insightsengineering/nesttemplate
```

Comment:
hey @shajoezhu @Melkiades 
this one is redundant - can you revert it please?

Reply:
yes will do!